package.json

```diff
@@ -3,7 +3,7 @@
   "dependencies": {
     "bootstrap": "3.4.1",
     "docsearch.js": "2.4.1",
-    "jquery": "3.4.0",
+    "jquery": "^3.5.1",
```

Comment:
You upgraded the dependencies on purpose?

Reply:
No, I don't know why I committed these updates... I did a `npm audit fix` after the `npm install`, it's probably this that updated dependencies. I will remove this file from my commit